egs/aishell/s10/chain/inference.py

```diff
@@ -47,7 +47,14 @@ def main():
     specifier = 'ark,scp:{filename}.ark,{filename}.scp'.format(
         filename=os.path.join(args.dir, 'confidence'))
 
-    writer = kaldi.CompressedMatrixWriter(specifier)
+    if args.save_as_compressed == 1:
```

Comment:
Hm, it doesn't seem super Pythonic to have 0/1 instead of a boolean.  I know reading boolean args in argparse is a pain, but there's a way to do it...

Reply:
in the function `_check_inference_args `
```python
args.save_as_compressed = (args.save_as_compressed == 1)
```
to change it from `int` to `bool`
would this be fine ?